frontend/scripts/code_generation/freezed/generate_freezed.sh

```diff
@@ -26,28 +26,28 @@ fi
 dart run build_runner clean && dart run build_runner build -d
 echo "Done generating files for appflowy_flutter"
 
-echo "Generating files for packages"
-cd packages
-for d in */; do
-  # Navigate into the subdirectory
-  cd "$d"
-
-  # Check if the subdirectory contains a pubspec.yaml file
-  if [ -f "pubspec.yaml" ]; then
-    echo "Generating freezed files in $d..."
-    echo "Please wait while we clean the project and fetch the dependencies."
-    if [ "$no_pub_get" = false ]; then
-      flutter packages pub get >/dev/null 2>&1
-    fi
-    dart run build_runner clean && dart run build_runner build -d
-    echo "Done running build command in $d"
-  else
-    echo "No pubspec.yaml found in $d, it can\'t be a Dart project. Skipping."
-  fi
-
-  # Navigate back to the packages directory
-  cd ..
-done
+# echo "Generating files for packages"
+# cd packages
+# for d in */; do
+#   # Navigate into the subdirectory
+#   cd "$d"
+
+#   # Check if the subdirectory contains a pubspec.yaml file
+#   if [ -f "pubspec.yaml" ]; then
+#     echo "Generating freezed files in $d..."
+#     echo "Please wait while we clean the project and fetch the dependencies."
+#     if [ "$no_pub_get" = false ]; then
+#       flutter packages pub get >/dev/null 2>&1
+#     fi
+#     dart run build_runner clean && dart run build_runner build -d
+#     echo "Done running build command in $d"
+#   else
+#     echo "No pubspec.yaml found in $d, it can\'t be a Dart project. Skipping."
+#   fi
+
+#   # Navigate back to the packages directory
+#   cd ..
+# done
```

Comment:
Why this change?

Reply:
It's unnecessary to generate the freezed files in the packages because we didn't import freezed on them, and this command really takes time.